### PR TITLE
Remove AMP URL API Wrapper

### PR DESCRIPTION
**Pull request by sebastianbenz**
* no longer needed as API now supports CORS
* rename view folder -> iframe
* increase result view size to match default result
* API key no longer needs to be encrypted as its access controlled via request host

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@kul3r4 @juliantoledo PAL

File: api/server.go [hunk 1]
```diff
@@ -15,36 +15,13 @@
...
 import (
-	"encoding/json"
-	"google.golang.org/appengine"
 	"net/http"
 )
 
-// init the handlers
 func init() {
-	http.HandleFunc("/url", apiHandler)
+	http.HandleFunc("/", serveNotFound)
```
Comment (kul3r4) [inline api/server.go]: Small typo: serveNotFound -> serverNotFound

Comment (sebastianbenz) [inline api/server.go]: serveNotFound is the intended name => serve the notFoundPage.

**Comment by kul3r4** [comment]
LGTM.

**Pull request merged**